source/proxysql-conf.rst

```diff
@@ -1,5 +1,40 @@
-Configuring ProxySQL
-======================
+.. proxysql-conf::
+
+Configuring Load Balancing with ProxySQL
+========================================
+
+Percona XtraDB Cluster Operator provides choice of two cluster components to
+carry on load balancing and proxy service: you can use either `HAProxy<https://haproxy.org>`_ or `ProxySQL <https://proxysql.com/>`_.
+You can control which one to use, if any, enabling or disabling via the
+``haproxy.enabled`` and ``proxysql.enabled`` options in the ``deploy/cr.yaml``
+configuration file. 
+
+Use the following command to enable HAProxy:
+
+.. code:: bash
+
+   kubectl patch pxc cluster1 --type=merge --patch '{ \
+    "spec": {"haproxy":{ "enabled": false }, \
+    "proxysql":{ "enabled": true } \
+    }}'
+
+.. note:: For obvious reasons the Operator will not allow the simultaneous
+   enabling of both HAProxy and ProxySQL.
+
+The resulting setup will use zero PXC node (``cluster1-pxc-0`` by default)
+as writer. 
+
+When the cluster with ProxySQL is upgraded, the following steps
+take place. First, reader nodes are upgraded one by one: the Operator waits
+until the upgraded node shows up in ProxySQL with online status, and then
+proceeds to upgrade the next node. When the upgrade is finished for all reader
+nodes, then writer PXC node is finally upgraded.
```

Comment:
please `s/node/PXC member/` in all documents.
reason: when we say "node" -- user can think about K8s node

```suggestion
When the cluster with ProxySQL is upgraded, the following steps
take place. First, reader memebers are upgraded one by one: the Operator waits
until the upgraded node shows up in ProxySQL with online status, and then
proceeds to upgrade the next memeber. When the upgrade is finished for all reader
memeber, then the writer PXC member is finally upgraded.
```